pyuploadcare/api.py

```diff
@@ -196,6 +195,11 @@ def _request():
             else:
                 raise
 
+def generate_secure_signature(secret, expire):
```

Comment:
1. it shouldn't sit here. move to utils
2. both secret and expire should be optional and default to `conf.xxx`